src/components/App/App.js

```diff
@@ -6,6 +6,10 @@ import Landing from '../Pages/Landing';
 import MyProfile from '../Pages/MyProfile/MyProfile'; // Import the MyProfile component
 import ContactUs from '../Pages/ContactUs';
 import Create from '../Pages/Create';
+import CopyrigthsNotice from '../Footer/CopyrigthsNotice/CopyrigthsNotice';
+import TermsOfServices from '../Footer/TermsOfServices/TermsOfServices';
+import PrivacyPolicy from '../Footer/PrivacyPolicy/PrivacyPolicy'; 
```

Comment:
It appears your attempting to import these components, however they aren't in the PR. Double check to make sure you're pushing up all your changes.